src/MakeTravisYml.hs

```diff
@@ -1606,21 +1533,26 @@ configFieldDescrs =
         (error "we don't pretty print")
         (fmap Just PU.parseTokenQ)
         cfgProjectName
-        (\x cfg -> cfg { cfgProjectName = x })
+        (set #cfgProjectName)
     , PU.listField  "branches"
         (error "we don't pretty print")
         PU.parseTokenQ
         cfgOnlyBranches
-        (\x cfg -> cfg { cfgOnlyBranches = x })
+        (set #cfgOnlyBranches)
     , PU.simpleField  "folds"
         (error "we don't pretty print")
         (sepBy parseFoldQ (munch1 isSpace))
         (\cfg -> [\_ -> cfgFolds cfg])
         (\x cfg -> cfg { cfgFolds = foldl' (flip id) (cfgFolds cfg) x })
     , PU.boolField "ghc-head"
         cfgGhcHead
-        (\b cfg -> cfg { cfgGhcHead = b })
+        (set #cfgGhcHead)
```

Comment:
setting is as simple, but actually we need lenses to use parsec from `Cabal-2.4`.